content/_help/verify-your-identity/phone-number._en.md

```diff
@@ -23,14 +23,17 @@ We are most likely to successfully verify your identity if you enter your primar
 You can use a prepaid cell phone to verify your identity. However, some prepaid phone numbers contain risk factors that might cause you to fail identity verification.
 
 ## You cannot verify your identity with:
-- International numbers
+- International numbers and numbers in some U.S. territories
 - Google voice or similar VOIP (Voice Over Internet Protocol) numbers
```

Comment:
Completely off-topic to your story, but I have always seen this styled as "VoIP" because the "over" should be lowercase. Do we have a style guide conclusively saying the opposite?

Reply:
Not in the style guide, but I'll add this in! Let's spell it the way you mentioned with the lowercase. Nice catch!